FFWEB-1563: Added information about `channel` attribute in `ff-checkout-tracking` (#238)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/documentation/tracking-guide.md

   <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
</ff-checkout-tracking>
```
If you do so, you can search for the child elements by invoking the `checkoutTrackingElement.trackCheckoutItems()` function manually.

> If you want to send the tracking request to a channel different from the one configured in ff-communication, use the `channel` attribute.

Reviewer: Formatting is gone again.
```suggestion
If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
```

   <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
</ff-checkout-tracking>
```

> If you want to send tracking request to the different channel, use `channel` attribute

Reviewer 2: I am wondering: `to the different channel` vs `to a different channel`. The later option seems to be more accurate in this context. @omikron-fp: what do you make of it?

Reviewer: I'd go for a bit more details.

_If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute._

> If you want to send the tracking request to a channel different from the one configured in ff-communication, use the `channel` attribute.

Reviewer: For consistency:
```suggestion
If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute.
```
For even more consistency, there are several mentions of `ff-` elements in regular text that could be unwrapped from `< >`. But I'll leave that to you to decide.

```html
<ff-checkout-tracking disable-auto-tracking>

>    <ff-checkout-tracking-item record-id="10321926" count="5" channel="new-channel-name"></ff-checkout-tracking-item>

Reviewer 2: [OPTIONAL]
I'd replace `new-channel-name` with something meaningful or at least wrap it with square brackets to stress that it os only a placeholder for some channel name.

</ff-checkout-tracking>
```

> If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.

Reviewer 2: [OPITONAL]
I think I have already replaced `poke around` with some more official expression. Please, check if there are more occurrences in the documentation and replace if applicable. A good chance to improve it a little.

> If you do so, you can search for the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.

Reviewer: I think this line makes more sense at its original position before `If you want to send the tracking request` (L102).
And while at it, the sentence could need a little update:
```suggestion
If you do so, you can search for the child elements by invoking the `checkoutTrackingElement.trackCheckoutItems()` function manually.
```
